src/main/java/org/simplify4u/slf4jmock/mockito/LoggerAnnotationEngine.java

```diff
@@ -66,9 +67,21 @@ public AutoCloseable process(Class<?> clazz, Object testInstance) {
         };
     }
 
+    private static List<InstanceField> allDeclaredFieldsOf(Object testInstance) {
+        List<InstanceField> result = new ArrayList<>();
+        for (Class<?> clazz = testInstance.getClass();
+             clazz != Object.class;
+             clazz = clazz.getSuperclass()) {
```

Comment:
Pleas try to put in one line ... multiple line for `for` statement can be mislead 😄 
if line will be too long, we can try rename variable `testInstance` to something shorter